server/controllers/apply/applicationsController.ts

```diff
@@ -51,17 +51,14 @@ export default class ApplicationsController {
         const person = await this.personService.findByCrn(req.user.token, crn)
         const offences = await this.personService.getOffences(req.user.token, crn)
 
-        // TODO: For now, we treat the first offence as the index offence, going
-        // forward, we'll need to design an approach to select one if there are
-        // more than one
-        const offence = offences[0]
+        const offenceId = offences.length === 1 ? offences[0].offenceId : undefined
```

Comment:
I don't feel too strongly about it but if we're setting a variable to be falsy I prefer `null` over `undefined` as it makes things easier to debug if something goes wrong.

Reply:
Yeah, good call 👍